include/h2o/socketpool.h

```diff
@@ -61,27 +61,25 @@ typedef struct st_h2o_socketpool_target_t {
             socklen_t len;
         } sockaddr;
     } peer;
+    void *data_for_balancer;
 
     struct {
         h2o_linklist_t sockets;
+        size_t request_count;   /* synchoronus operations should be used, counting requests on the fly */
```

Comment:
What does this value represent? Does it show the number of total connections to the target? Or does it show the number of leased (i.e. non-pooled) connections?

I would appreciate it if could rename the property that clarifies my questions. Also, I think it would be preferable to not use _request_ as part of the name here, since socketpool just pools connections (and leases them). It does not know if a socket is used for "requests."